src/Billable.php

```diff
@@ -29,6 +29,7 @@
 trait Billable
 {
     use PopulatesMollieCustomerFields;
+    use ManagesInvoices;
```

Comment:
One-off payment related methods should go in a separate trait. Despite the invoice word it's really different functionality. How about ManagesOneOffPayments?